test/osb_test/osb_test.go

```diff
@@ -209,4 +209,26 @@ var _ = Describe("Service Manager OSB API", func() {
 			})
 		})
 	})
+
+	Describe("Get Last Operation", func() {
+
+		assertLastOperationHasState := func(url string) {
+			resp := ctx.SMWithBasic.GET(url).WithHeader("X-Broker-API-Version", "oidc_authn.13").
+				Expect().Status(http.StatusOK).JSON().Object()
+
+			resp.ContainsKey("state")
+		}
+
+		Context("For service instance", func() {
+			It("should return state", func(){
+				assertLastOperationHasState(validBroker.OSBURL+ "/v2/service_instances/iid/last_operation")
```

Comment:
Can you also add test requests with appropriate query parameters like described here: https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md#parameters